メインプログラムからテストデータを分離

sprngTst001 opened the pull request:
テストデータセットアップを行うアノテーションの追加
テストデータ定義ファイルを追加
fix #16

thayashi131050 commented on src/test/resources/dbunit/expectedDataset1.xml:
```diff
@@ -4,4 +4,4 @@
     <peopleoutput person_id="4" full_name="DEFAAA"/>
-</dataset>
\ No newline at end of file
+</dataset> 
```
不要な空白で差分が発生しています。
`schema-all.sql`の指摘と一緒に対応お願いします。

thayashi131050 reviewed (changes requested):
`schema-all.sql`をtestに移動したことで、mainのプログラムで実行のたびに出力テーブルのデータが増えていく状態になっています。mainとtestはそれぞれ以下の役割で作成してください。
- main
  - `peopleInput`にあるデータを加工して`peopleOutput`に出力する
    - `peopleInput`のデータは事前に用意されている前提
  - `peopleOutput`は実行のたびに初期化する
    - 実行後は`peopleInput`に対するデータのみが`peopleOutput`に存在することを保証
- test
  - `peopleInput`のデータを用意する
  - `peopleOutput`の内容を検証する

また、`fix #16`は目的が異なるのでコミットを分けてください。
#17 に対するコミットには`close #17`を含めてください。

sprngTst001 committed:
fix #16

sprngTst001 committed:
close #17

mainプログラムでInputテーブルの初期化を行わないよう変更(事前に用意されている前提)

testプログラムでInputテーブルの初期化・テストデータ作成を行うよう変更

sprngTst001 commented:
今まで取消して再度プルリクエストを投げていましたが、修正するだけでここに反映されるのですね...。分かりづらくしていたようで申し訳ないです。
